ci-operator/config/maistra/envoy/maistra-envoy-maistra-2.2.yaml

```diff
@@ -76,6 +76,30 @@ tests:
           cpu: 100m
           memory: 100Mi
       timeout: 8h0m0s
+    - as: envoy-update-proxy-2.2
+      cli: latest
+      commands: |
+        ./tools/automator-main.sh \
+        -o maistra \
+        -f /creds-github/github-token \
```

Comment:
for this cred to work you will need something like https://github.com/openshift/release/blob/master/ci-operator/config/maistra/test-infra/maistra-test-infra-main.yaml#L42-L44

Reply:
Done.